task-management/src/main/java/org/exoplatform/task/management/controller/UserController.java

```diff
@@ -69,15 +69,28 @@
     @Resource
     @Ajax
     @MimeType.JSON
-    public Response findUser(String query) throws Exception { // NOSONAR
+    public Response findUser(String query, String projectName) throws Exception { // NOSONAR
       ListAccess<org.exoplatform.task.model.User> list = userService.findUserByName(query);
       JSONArray array = new JSONArray();
       for(org.exoplatform.task.model.User u : list.load(0, UserUtil.SEARCH_LIMIT)) {
-        JSONObject json = new JSONObject();
-        json.put("id", u.getUsername());
-        json.put("text", u.getDisplayName());
-        json.put("avatar", u.getAvatar());
-        array.put(json);
+          JSONObject json = new JSONObject();
+          String username = u.getUsername();
+          SpaceService sService =  CommonsUtils.getService(SpaceService.class);
+          Space space = sService.getSpaceByPrettyName(projectName);
+          if (space != null) {
+              String[] spaceMembers = space.getMembers();
```

Comment:
use use org.exoplatform.social.core.space.impl.SpaceServiceImpl#isMember(org.exoplatform.social.core.space.model.Space, java.lang.String) to check if user is member if space